docs/versioning.md

```diff
@@ -1,55 +1,46 @@
-# Release compatibility
+# Release versioning
 
 Modules are updated regularly. The changes adhere to [semantic versioning](https://semver.org/), with releases labeled as `{major}.{minor}.{patch}`. [Git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) are used to identify module versions.
 
-A major release level indicates a change that is not compatible with earlier versions. A minor release includes new capabilities and includes only compatible changes. A patch includes mostly bugs fixes that are compatible with an earlier version.
+Release levels apply to both modules and submodules. For example, a major version change to a submodule results in a major release for the whole module. However, changes to examples do not drive release levels, unless the example is published as a Deployable Architecture to the IBM Cloud.
 
-Release levels apply to both modules and submodules. For example, an incompatible change to a submodule results in a major release for the whole module. However, changes to examples do not drive release levels. So, a minor release to the module might include incompatible changes in example code.
+## Determining version updates
 
-## Determining compatibility
+No universal criteria exist to determine terraform module versioning. However, when we evaluate changes, we consider major version updates only necessary when any of the actual infrastructure that is deployed by the module will either be destroyed or modified in a way that may affect every day operations.
 
-No universal criteria exist to determine compatibility. However, when we evaluate changes, we consider the possible assumptions that client developers might have and that indicate that a particular change is likely not compatible.
+### Major version changes
 
-### Incompatible changes
+A major version release is one that impacts the actual infrastructure that is deployed by the module itself. The following are example of major version updates:
 
-At a high level, a change that is incompatible with earlier versions is a change that causes code to stop working or might destroy infrastructure. Sometimes modules require these incompatible changes. Known incompatible changes are introduced only in major releases.
+- Adding changes that may destroy infrastructure or update resources (that may impact every day operations) that were created in the previous version
```

Comment:
I'd split the bullet here - put the update separately along with "that may impact every day operations" that apply only to updates